tokio/src/net/tcp/listener.rs

```diff
@@ -216,11 +216,13 @@ impl TcpListener {
     ///
     /// # Panics
     ///
-    /// This function panics if thread-local runtime is not set.
+    /// This function panics if it is not called from within a runtime with
+    /// IO enabled or if the thread-local runtime is not set.
```

Comment:
```suggestion
    /// This function panics if it is not called from within a runtime with
    /// IO enabled.
```

Reply:
@hds Did you have any thoughts on this doc change?

Reply:
@Darksonn I agree with them, it fits in with the rest of the docs. I'll make them in all the relevant spots in the next few days - I've been on holiday and have extended that to not opening my computer at all for a week. (-;

Reply:
Alright. The next release is probably a month from now, so there's no rush.

Reply:
Fixed.